src/renderer/src/lib.rs

```diff
@@ -12,15 +12,14 @@ extern crate mopa;
 
 extern crate glutin;
 extern crate cgmath;
-
-extern crate amethyst_ecs;
+extern crate specs;
 
 /// Contains the included Render Targets
 pub mod target;
 /// Contains the included Passes
 pub mod pass;
 
-use amethyst_ecs::{Component, VecStorage};
+use self::specs::{Component, VecStorage};
```

Comment:
It is done in order to have a `Light` `Component` without a wrapper type (the way it was implemented before context rewrite PR), since it isn't possible to impl traits for types defined in an outside crate.

So it is either an `amethyst::ecs::components::Light` `Component`, which wraps a `amethyst_renderer::Light` and no `specs` import in `amethyst_renderer` or a plain `amethyst_renderer::Light` `Component` with a `specs` import.

I guess returning back to a wrapper type would work, but it is less convenient to access the `Light` `Component` this way. Instead of getting say `light.color` you would have to get `light.light.color`.